src/RemediationRoleSaver.cpp

```diff
@@ -41,3 +62,21 @@ QString RemediationSaverBase::guessFilenameStem()
     // TODO: Add guess that uses benchmark and profile names
     return QString("remediation");
 }
+
+
+BashRemediationSaver::BashRemediationSaver(QWidget* parentWindow, ScanningSession* session):RemediationSaverBase(parentWindow, session)
+{
+    mSaveMessage = QObject::tr("Save remediation role as a bash script");
+    mFiletypeExtension = "sh";
+    mFiletypeTemplate = QObject::tr("bash script (*.%1)");
+    mFixTemplate = QObject::tr("urn:xccdf:fix:script:sh");
```

Comment:
any reason why this is not initialized in the initialization list?

Reply:
It belongs to the base class and the base class doesn't know how should the content be.
Having it in the cpp file like this has the added benefit that If there is a new format of remediation rules, the person who will implement it will immediately know what is the meaning of each individual string thanks to the fact that it is assigned to the variable as opposed to being specified as "the 3rd argument to the constructor".

Reply:
ah, ok, I didn't realize you are setting parent class members in the ctor